content/doc/book.html.haml

```diff
@@ -3,18 +3,17 @@ layout: simplepage
 title: "Jenkins Handbook"
```

Comment:
Could we move this into `/doc/book/index.html.haml`?  It seems strange that it is sitting outside.